misc/drone-scripts/validate-and-serialize.sh

```diff
@@ -12,15 +12,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+DS='-ds=datastore1'
 unit_test_array=($TEST_URL_ARRAY)
 numServers=${#unit_test_array[@]}
 DRONE_BUILD_NUMBER=${DRONE_BUILD_NUMBER:=0}
 prevBuildStatus=`drone build info vmware/docker-volume-vsphere $(( $DRONE_BUILD_NUMBER-$numServers ))`
 outArray=($prevBuildStatus)
 total_builds=`drone build list vmware/docker-volume-vsphere| wc -l`
 
-govc datastore.mkdir docker-volume-vsphere/$DRONE_BUILD_NUMBER
+govc datastore.mkdir $DS docker-volume-vsphere/$DRONE_BUILD_NUMBER
```

Comment:
This change is not from me. It was made by Ritesh in master, I rebase my change with master and then you see the change. I have removed from my commit log.